SpatialGDK/Extras/schema/rpc_components.schema

```diff
@@ -3,17 +3,24 @@ package unreal;
 
 import "unreal/gdk/core_types.schema";
 
+type TracePayload {
+    bytes trace_id = 1;
+    bytes span_id = 2;
+}
+
 type UnrealRPCPayload {
     uint32 offset = 1;
     uint32 rpc_index = 2;
     bytes rpc_payload = 3;
+    option<TracePayload> rpc_trace = 4;
```

Comment:
How are we ensuring these aren't duplicated? Does the worker sdk do this for us?

Reply:
Discussed offline